geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/aggregators/HBaseArrowAggregator.scala

```diff
@@ -43,8 +38,12 @@ object HBaseArrowAggregator {
                 stats: GeoMesaStats,
                 filter: Option[Filter],
                 ecql: Option[Filter],
-                hints: Hints): (Map[String, String], QueryPlan.Reducer) = {
+                hints: Hints): (Map[String, String], FeatureReducer) = {
     val conf = ArrowScan.configure(sft, index, stats, filter, ecql, hints)
     (conf.config + (GeoMesaCoprocessor.AggregatorClass -> classOf[HBaseArrowAggregator].getName), conf.reduce)
   }
+
+  class HBaseArrowResultsToFeatures extends ArrowResultsToFeatures[Array[Byte]] {
+    override protected def bytes(result: Array[Byte]): Array[Byte] = result
```

Comment:
Some over-refactoring idea:)  Maybe there could be a trait for this?

Reply:
It's so trivial, I'd lean towards not...